interfaces/cython/cantera/_onedim.pxd

```diff
@@ -69,13 +69,19 @@ cdef extern from "cantera/oneD/StFlow.h":
     cdef cppclass CxxStFlow "Cantera::StFlow" (CxxDomain1D):
         void setTransportModel(const string&) except +translate_exception
         void setTransport(CxxTransport&) except +translate_exception
+        string type()
         string transportModel()
         void setPressure(double)
         void enableRadiation(cbool)
         cbool radiationEnabled()
         double radiativeHeatLoss(size_t)
         double pressure()
         void setFixedTempProfile(vector[double]&, vector[double]&)
+        size_t getSolvingStage()
+        void setSolvingStage(size_t)
+        void solveElectricField()
+        void fixElectricField()
+        cbool doElectricField(size_t)
```

Comment:
Most of these need to be marked with `except +translate_exception`, since `StFlow` throws `NotImplementedError`.